pyatlan/model/search.py

```diff
@@ -1829,6 +1827,25 @@ def validate_query(cls, v, values):
         else:
             raise ValueError("Either query or post_filter is required")
 
+    @validator("sort", always=True)
+    def set_sort(cls, sort):
+        missing_guid_sort = True
+        sort_by_guid = TermAttributes.GUID.value
+        auditsearch_sort_by_guid = "entityId"
+        searchlog_sort_by_guid = "entityGuidsAll"
+        # Check if the sort by GUID is included
+        for option in sort:
+            if option.field and option.field in (
+                sort_by_guid,
+                auditsearch_sort_by_guid,
+                searchlog_sort_by_guid,
+            ):
+                missing_guid_sort = False
+                break
+        if missing_guid_sort:
+            sort.append(SortItem(sort_by_guid))
```

Comment:
This still feels a bit strange — we're checking whether the sort includes a GUID-equivalent for indexsearch, auditsearch, and searchlog (all different), but then if that's missing we're only appending the sort by GUID (indexsearch) — the `sort_by_guid` variable here looks like it is hard-coded to the `TermAttributes.GUID.value` (?)